### DISCUSSION TITLE
Early Hints support

### DISCUSSION THREAD

**Discussion by rdgout** [laravel/framework#49388] [General]
I noticed in recent times that I've been seeing more information about early hints. Symfony has been supporting it for awhile. 
Are there any plans to support it in Laravel too? 

For reference: https://symfony.com/blog/new-in-symfony-6-3-early-hints

**Comment by indykoning** [comment]
This is currently only supported on Frankenphp.
And Implementing it is a bit of a hassle since you should know your headers BEFORE you've even started preparing.
But it's possible using middlewares and some caching.
In the meantime https://github.com/justbetter/laravel-http3earlyhints adds support.

**Reply by rdgout** [reply]
Cool man, thanks for your work 😄
